packages/slate-editor/src/modules/nodes-hierarchy/fixers/insertTableRow.ts

```diff
@@ -0,0 +1,23 @@
+import { Transforms } from 'slate';
+import type { NodeEntry } from 'slate';
+import type { Editor } from 'slate';
+
+import { createParagraph } from '#extensions/paragraphs';
+import { createTableCellNode, createTableRowNode } from '#extensions/tables';
+
+export function insertTableRow(editor: Editor, [node, path]: NodeEntry) {
+    Transforms.insertNodes(
+        editor,
+        [
+            createTableRowNode({
+                children: [createTableCellNode({ children: [createParagraph()] })],
+            }),
+        ],
```

Comment:
This is duplicating already existing code to construct rows/cells: `TableRowNode.createTableRow()`. 

I'd better expose `createTableRow` on the `TablesEditor` and use it here.

```suggestion
        TablesEditor.createTableRow(),
```

```diff
packages/slate-tables/src/nodes/TableRowNode.ts
===================================================================
@@ -20,7 +20,7 @@
             children:
                 typeof children === 'number'
                     ? Array.from(Array(children)).map(() => TableCellNode.createTableCell(editor))
-                    : children ?? [],
+                    : children ?? [TableCellNode.createTableCell(editor)],
         });
     }
```

Reply:
#269